### PR TITLE
IGNITE-13501 AssertionError: Invalid value in testMergeServersFail1_8

### PR DESCRIPTION
**Pull request by vldpyatkov**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/OnePhaseCommitAndNodeLeftTest.java [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+import org.apache.ignite.transactions.TransactionRollbackException;
+import org.junit.Test;
+
+import static org.apache.ignite.internal.IgniteNodeAttributes.ATTR_IGNITE_INSTANCE_NAME;
+
+/**
+ * Node left a topology when a one phase transaction committing.
+ */
+public class OnePhaseCommitAndNodeLeftTest extends GridCommonAbstractTest {
+
```
Comment (ascherbakoff) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/OnePhaseCommitAndNodeLeftTest.java:50]: Empty line not needed.

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/OnePhaseCommitAndNodeLeftTest.java [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+        stopAllGrids();
+    }
+
+    /**
+     * Tests an implicit transaction on a cache without backups.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testImplicitlyTxZiroBackups() throws Exception {
```
Comment (ascherbakoff) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/OnePhaseCommitAndNodeLeftTest.java:80]: Ziro -> Zero

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/OnePhaseCommitAndNodeLeftTest.java (hunk 1 above)
Comment (vldpyatkov) [reply modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/OnePhaseCommitAndNodeLeftTest.java:50]: Fixed.

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/OnePhaseCommitAndNodeLeftTest.java (hunk 2 above)
Comment (vldpyatkov) [reply modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/OnePhaseCommitAndNodeLeftTest.java:80]: Fixed.

**Pull request closed without merging**